src/Microsoft.Extensions.Configuration.Binder/project.json

```diff
@@ -20,7 +20,11 @@
     "Microsoft.Extensions.Configuration": "1.0.0-*"
   },
   "frameworks": {
-    "net451": {},
+    "net451": {
```

Comment:
System.ComponentModel.TypeConverter targets dotnet5.1. If I update this project to target netstandard1.1 (and remove net451) it restores and compiles just fine. However when I restore the corresponding test project, it fails:
```
Errors in D:\work\katana\Configuration\test\Microsoft.Extensions.Configuration.Binder.Test\project.json
    System.Runtime 4.0.20 provides a compile-time reference assembly for System.Runtime on .NETFramework,Version=v4.5.1, but there is no run-time assembly compatible with win7-x64. 
```